convene-web/app/views/guides/_rooms.md

```diff
@@ -0,0 +1,132 @@
+- [Entering Rooms](#entering-rooms)
+  - [Room Links](#room-links)
+- [Room Privacy and Security](#room-privacy-and-security)
+  - [Hidden Rooms](#hidden-rooms)
+  - [Private Rooms](#private-rooms)
+  - [Locked Rooms](#locked-rooms)
+    - [How To Lock a Room](#how-to-lock-a-room)
+    - [Access Codes](#access-codes)
+    - [Some Caveats](#some-caveats)
+
+[Rooms] are the building blocks of a [Space]. Think of them just like [Rooms] in
+a house or an office. You can [lock them](#locked-rooms) or
+[leave them wide open](#unlocked-rooms). You can even
+[hide them like a secret passageway](#hidden-rooms)!
+
+Most [Rooms] have a specific purpose:
+
+- A kitchen where you store your favorite recipes or publish your food pictures;
+- a studio where you show off your art (and maybe take commissions or offer
+  originals and prints for sale);
+- an office for project materials and updates; or
+- a front porch where you broadcast your stream-of-consciousness thoughts to the
+  world and catch up with your neighbors and acquaintances.
+
+To tune each [Room] for its purpose, fill it with [Furniture] that facilitates
+activities or collect and share information.
```

Comment:
Maybe a little clearer: ...[Furniture] that facilities activities, such as collecting or sharing information.

Reply:
Yea it's an awkward sentane. Gonna remake.